client/src/components/App.jsx

```diff
@@ -1,7 +1,84 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+import Photo from './Photo.jsx';
 
-const App = () => (
-  <h1>hello world</h1>
-);
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentRoom: null,
+      photos: [],
+      view: 'main',
+      hasMounted: false,
```

Comment:
Is there utility in having a hasMounted bool when fetchPhotos is only called after the component mounts?

Reply:
good point will factor this out to handle a 'loading' view before component mounts.